images/base/Dockerfile.ppc64le

```diff
@@ -0,0 +1,18 @@
+#
+# This is the base image from which all OpenShift Origin images inherit. Only packages
+# common to all downstream images should be here. Depends on Centos 7.2+.
+#
+# The standard name for this image is openshift/origin-base
+#
+FROM ppc64le/centos:centos7
```

Comment:
@smarterclayton  I agree with that, my biggest concern is - we should have different images for different architectures like `openshift/origin-pod-amd64` for amd and `openshift/origin-pod-ppc64le` or we should create manifest image `openshift/origin-pod` which will point to all architecture images?